qiskit/circuit/library/standard_gates/x.py

```diff
@@ -570,6 +569,27 @@ def inverse(self):
     #                         [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 0],
     #                         [0, 0, 0, 0, 0, 0, 0, 1, 0, 0, 0, 0, 0, 0, 0, 0]], dtype=complex)
 
+class C3XGate(C3RXGate):
+    def __init__(self, angle=None, label=None, ctrl_state=None):
+        import warnings
+        if angle == numpy.pi / 8:
+            warnings.warn('The parameter angle in C3XGate is going to be removed in the future. '
+                          'When angle=pi/8, use C3SqrtXGate()',
+                          DeprecationWarning, stacklevel=2)
+        elif angle == numpy.pi / 4:
+            warnings.warn('The parameter angle in C3XGate is going to be removed in the future. '
+                          'angle=pi/8, while be the default behavior C3XGate.',
+                          DeprecationWarning, stacklevel=2)
+        elif angle is not None:
+            warnings.warn('The parameter angle in C3XGate is going to be removed in the future. '
+                          'When angle!=pi/8, use C3RXGate()',
+                          DeprecationWarning, stacklevel=2)
+
+        super().__init__(angle=angle or numpy.pi/4, label=label, ctrl_state=ctrl_state)
```

Comment:
Does this work if angle is 0? Maybe we need a proper `is None` comparison

Reply:
good catch. Fixed in a3800b8ca